src/store/helpers/index.js

```diff
@@ -36,8 +36,8 @@ export const createBiobankRSQLQuery = (state) => transformToRSQL({
   operands: flatten([
     createInQuery('country', state.filters.selections.country || []),
     createInQuery('id', state.biobankIdsWithSelectedQuality),
-    createInQuery('network', state.filters.selections.biobank_network || []),
-    createComparisons('covid19biobank', state.filters.selections.covid19 || [])
+    createQueryWithSatisfyAll(state.filters.selections.biobank_network, 'network', state.filters.satisfyAll.biobank_network),
+    createQueryWithSatisfyAll(state.filters.selections.covid19, 'covid19biobank', state.filters.satisfyAll.covid19)
```

Comment:
it looks like satisfy all replaces the original selection?

Reply:
Something like

createQuery(state.filters.selection.covid19, 'covid19biobank' , state.filters.satisfyAll.includes('covid19'))

Reply:
Resolved in 
1d0b578